src/modules/email/domain/constants/injection-tokens.ts

```
@@ -0,0 +1 @@
+export const MAIL_SERVICE_TOKEN = 'MAIL_SERVICE_TOKEN';
```

ESLint

[error] Insert `⏎`

(prettier/prettier)